src/common/calendar/_timePickerCalendar.scss

```diff
@@ -33,3 +33,84 @@
   }
 }
 
+
+@media (max-width: $screen-xs-max) {
+  .fc-time-grid .fc-slats td {
```

Comment:
Is there a way to assign custom classNames instead of targeting the existing ones?

Reply:
I think i can wrap them under `.app-Calendar` parent is the best?